source/manual/create-a-local-transaction.html.md

```diff
@@ -0,0 +1,236 @@
+
+Once you've found the service, gather the following information:
+
+* An [LGSL_CODE](#lgsl-code) - the ID of the service in ESD
+* An [LGIL_CODE](#lgil-code)
+* A [SERVICE_NAME](#seek-content-design-advice)
+* A [SERVICE_DESCRIPTION](#seek-content-design-advice)
+* The [SERVICE_SLUG](#seek-content-design-advice)
+* The [PROVIDING_TIER](#providing-tier)
+* Which (if any) nations the service is unavailable in?
+
+__Note__: In the following, wherever you see the name of a variable (i.e. LGSL_CODE) in a terminal command or code snippet, please substitute the actual value.
+
+## Get a list of local authority slugs and URLs
+
+We need to create a CSV file containing the `name`, `slug` and an empty space for a `url` for all available local authorities currently in LLM.
+
+The service product owner (your PM or DM should be able to identify them for you) can then add the specific URLs for each local authority that offers the service and return the finished file to us for import. This is used to associate the URL with the service and the local authority in LLM.
+
+Log into the LLM console...
+
+```bash
+$ gds govuk connect app-console -e integration local-links-manager
+```
+
+Make a note of the `MACHINE_NO` that you are connected to.
+
+Then in the rails console...
+
+```ruby
+file = "#{Rails.root}/tmp/LGSL_CODE.csv"
+```
+
+Make a note of the file path, it should look something like this...
+
+`/data/vhost/local-links-manager/releases/YYYYMMDDHHMMSS/tmp/LGSL_CODE.csv`
+
+Generate the CSV file and exit out of the rails console.
+
+```ruby
+CSV.open(file, 'w') do |csv|
+  csv << ["name", "slug", "url"]
+  LocalAuthority.order(name: :asc).each do |la|
+    csv << [la.name, la.slug, nil]
+  end
+end
+```
+
+Download the CSV file just you just created to your local machine...
+
+```bash
+$ gds govuk connect scp-pull -e integration backend:MACHINE_NO REMOTE_FILE_PATH LOCAL_FILE_PATH
+```
+
+Hand this over to the service product owner.
+
+_The `name` is only present in the CSV file to assist the product owner complete the URLs as differences between the local authority `name` and `slug` can make it tricky to match the local authority correctly. It is not used when importing the links into LLM._
+
+## Add the service to Publisher
+
+Create a PR that adds the following to the bottom of [this](https://github.com/alphagov/publisher/blob/a656ea44d0e3a086b096ff2b2053290f1b9fb4d9/data/local_services.csv) file:
+
+| LGSL | Description | Providing Tier |
+| ---- | ---- | ---- |
+| [LGSL_CODE](#lgsl-code) | [SERVICE_DESCRIPTION](#seek-content-design-advice) | [PROVIDING_TIER](#providing-tier) |
+
+Once your PR is merged and deployed, run [this rake task](https://github.com/alphagov/publisher/blob/main/lib/tasks/local_transactions.rake) in Publisher...
+
+```bash
+bundle exec rails local_transactions:fetch_and_clean
+```
+
+## Make the service unavailable for a nation in Frontend
```

Comment:
Soon this will be available to do in publisher - https://trello.com/c/hsAegPqf/311-epic-local-look-up-enable-publishers-to-control-the-regions-a-local-look-up-applies-to-and-manage-different-result-pages-copy, but for now this is right

Reply:
Yeah, I did see that card but thanks for reminding me about it.  Can't wait this current process is a bit hacky!